Release notes for CCM (#779)
harness/developer-hub: release-notes/cloud-cost-management.md

Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
:::

## February 20, 2023

### What's new

* AutoStopping Proxy for HTTPS and TCP connections.

 Harness CCM introduces **AutoStopping Proxy** to support AutoStopping for HTTPS and TCP connections. For more information, go to [Add load balancers](https://developer.harness.io/docs/category/add-load-balancer-for-autostopping-rules) and [Create AutoStopping rules](https://developer.harness.io/docs/category/create-autostopping-rules).

### Early access

  This release does not include early access features.

### Fixed issues

> *  The potential monthly savings displayed on the UI did not match with the Spot or On-Demand recommendations. (CCM-10698)

Reviewer: ```suggestion
* The potential monthly savings displayed on the UI did not match with the Spot or On-Demand recommendations. (CCM-10698)
```

>   The logic to calculate the potential monthly cost displayed on the UI has been fixed. Now, the savings match with the Spot or On-Demand recommendations.

Reviewer: ```suggestion
The logic to calculate the potential monthly cost displayed on the UI has been fixed. Now, the savings match with the Spot or On-Demand recommendations.
```

* Added the missing instance family types for Azure node pool recommendations. (CCM-10246)

* When you create a budget with an invalid **Period Starts from** date with respect to the **Budget Period**, the error message displayed was unclear - "Invalid request: Error in create budget operation. Start time of budget is invalid." (CCM-10487)

>  Now, the message is changed to "Invalid request: Budget Period and Period Start date cannot add up to be in the past."

Reviewer: ```suggestion
 Now, the message clarifies why the date is invalid - "Invalid request: Budget Period and Period Start date cannot add up to be in the past."
```

Suggestion to clarify what is changed about the message.
